spec/es_spec_helper.rb

```diff
@@ -59,7 +59,12 @@ def default_mapping_from_mappings(mappings)
   end
 
   def field_properties_from_template(template_name, field)
-    mappings = @es.indices.get_template(name: template_name)[template_name]["mappings"]
+    template = get_template(@es, template_name)
+    mappings = if ESHelper.es_version_satisfies?(">=8")
```

Comment:
suggestion: maybe for consistency we could also have a `get_mapping` method similar to `get_template`?